test/unit/linalg/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+################################################################################
+# Test utilities
+
+include(test_util)
+
+################################################################################
+# tests
+
+create_standard_cxx_test(NAME test_tridiag SOURCES tridiag.cpp) 
```

Comment:
Based on the discussion with @K20shores , you should change the name from `test_tridiag` to `tridiag` so that you do not get `test_test_` shown up in the testing later.